### ISSUE TITLE
Display custom color on card

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#4675]
If you set a custom color of a Project (or Collective/Event/etc) I think the preview card for that should be that color.

E.g. this Project has a green color:
<img width="1250" alt="Screen Shot 2021-09-14 at 6 53 01 PM" src="https://user-images.githubusercontent.com/5498878/133209578-fa0d5d99-5c04-49a2-8f00-4899569b08e4.png">

But it's still blue when you view it on the Collective page or the Projects page:
<img width="1249" alt="Screen Shot 2021-09-14 at 6 51 50 PM" src="https://user-images.githubusercontent.com/5498878/133209619-c104e6bf-f4d4-41e7-b8e1-6d7c6e7fabe7.png">

(Note, if you set a custom cover image it does show on the card so I think it would be a consistent design choice to all show the color if there is no custom cover image).

**Comment by Betree** [comment]
I agree that the project's primary color should be used to generate the card hero. The same should apply to events and connected collectives.

> if you set a custom cover image it does show on the card

It seems to be displayed on the example you shared:
![image](https://user-images.githubusercontent.com/1556356/133209940-d5d34a58-d7c0-43d8-a055-372f3e6eb0b0.png)
